packages/client/vite.config.ts

```diff
@@ -1,32 +1,41 @@
 import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
 
 export default defineConfig({
+  plugins: [react()],
   root: "src",
-  build: {
-    outDir: "../dist",
-    emptyOutDir: true,
-    sourcemap: true,
-    assetsInlineLimit: 0,
-    target: "es2022",
-  },
-  preview: {
-    port: 3000,
-  },
-  resolve: {
-    dedupe: ["proxy-deep", "styled-components"],
-  },
-  define: {
-    global: "globalThis",
-  },
-  optimizeDeps: {
-    esbuildOptions: {
-      target: "es2022",
-    },
-    exclude: ["@latticexyz/noise", "buffer"],
-  },
   server: {
+    port: 3000,
     fs: {
       strict: false,
     },
   },
+  // root: "src",
+  // build: {
+  //   outDir: "../dist",
+  //   emptyOutDir: true,
+  //   sourcemap: true,
+  //   assetsInlineLimit: 0,
+  //   target: "es2022",
+  // },
+  // preview: {
+  //   port: 3000,
+  // },
+  // resolve: {
+  //   dedupe: ["proxy-deep", "styled-components"],
+  // },
+  define: {
+    global: "globalThis",
+  },
+  // optimizeDeps: {
+  //   esbuildOptions: {
+  //     target: "es2022",
+  //   },
+  //   exclude: ["@latticexyz/noise", "buffer"],
+  // },
+  // server: {
+  //   fs: {
+  //     strict: false,
+  //   },
+  // },
 });
```

Comment:
@alvrs do you have a sense for which of these config options we still use/need? esp when it comes time to replace parcel

Reply:
at least `build` and `preview`. I think `resolve` and `optimizeDeps` were used to solve some bunding issue we ran into at some point, but we should probably approach approach the bunding situation with fresh eyes soon (given we can't even use vite for building for prod right now when mud is not linked locally). I believe `server` was used to fix an issue where vite complained it doesn't have access to the locally linked MUD (although it seems to work for me right now)

Reply:
all fixed! https://github.com/latticexyz/opcraft-readonly/pull/3/commits/9cd6fbb159dfa7bbffb439cff8e49928ae75539e